code/src/components/TodoList.js

```diff
@@ -0,0 +1,197 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+
+import styled from 'styled-components/macro'
+import { AddCross, ListBtn, OuterWrapper } from 'GlobalStyle'
+
+// Reducers
+import todo from 'reducers/todo'
+
+// Components
+import xBtn from '../assets/icon_cross.png'
+import check from '../assets/icon_check_hover.svg'
+import { AddTodo } from './AddTodo'
+// import { TodoCounter } from './TodoCounter'
+// import TaskCounter from './TaskCounter'
+import { CompleteAll } from './CompleteAll'
+import { DeleteAll } from './DeleteAll'
+
+export const TodoList = () => {
+  const items = useSelector((store) => store.todo.items)
+  const todosCounter = items.filter((item) => item.isCompleted === true)
+
+  const dispatch = useDispatch();
+
+  // -------dispatch reducers-------
+  const onToggleTodo = (id) => {
+    dispatch(todo.actions.toggleTodo(id))
+  }
+
+  const onDeleteTodo = (id) => {
+    dispatch(todo.actions.deleteTodo(id))
+  }
+
+  return (
+    <OuterWrapper>
+      {/* flex-start section */}
+      {' '}
+      <AddTodo />
+      <TodoCounter>
+        <h2>Completed: {todosCounter.length} / {items.length} </h2>
+      </TodoCounter>
+      <InnerWrapper>
+        {items.map((item) => (
+          <TaskContainer key={item.id}>
+            <TaskContainerWrapper>
+              <Wrapper>
+                <input
+                  type="checkbox"
+                  tabIndex="0"
+                  name="tasks"
+                  checked={item.isCompleted}
+                  onChange={() => onToggleTodo(item.id)} />
+                <InputText className={`${item.isCompleted ? 'completed' : ''}`}>{item.text}</InputText>
```

Comment:
Very nice to have the className do diffrent things depending on if completed or not, very smart :)